File: mario.c
#include <cs50.h>
#include <stdio.h>

int get_height (void);
int main(void)
{
    int x = get_height();
    for(int i=1; i<= x; i++)
    {
        int n = x - i;
        for(int j=1 ; j<= n ; j++)
        {
            printf(" ");
        }

        for(int k=1; k<=i ; k++)
        {
            printf("#");
        }
        printf("\n");
    }

}





int get_height (void)
{
    int x;
    do
    {
        x = get_int("Height: ");
    }
    while(!(x>=1 && x<=8));
    return x;
}

File: mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if 0 < height < 9:
        break

for i in range(1, height + 1):
    n = height - i
    for j in range(n):
        print(" ", end="")
    for k in range(i):
        print("#", end="")
    print("")
